src/main/java/com/hibernate/model/User.java

```diff
@@ -15,6 +17,16 @@
     private Long id;
     private String email;
     private String password;
+    @ManyToMany
+    private List<Role> roles;
```

Comment:
```suggestion
    private Set<Role> roles;
```
set suits better here because we don't want our roles to be repeated